system-tests/.env

```diff
@@ -0,0 +1,2 @@
+OTELCOL_IMG=otel/opentelemetry-collector-contrib-dev:latest
```

Comment:
where would the sys test run? If under git action maybe it will be cleaner to inject system vars using project settings